### ISSUE TITLE
mapValue function problem

### ISSUE THREAD

**Issue by haps-basset** [Bodmer/TFT_eWidget#2]
Hello
I cannot find reference of lib for mapValue function in   Analogue meter example.
Could you point or publish mapValue function code?
Haps

**Comment by Bodmer** [comment]
Sorry, seem to have missed it out. Here it is:

```
float mapValue(float ip, float ipmin, float ipmax, float tomin, float tomax)
{
  return tomin + (((tomax - tomin) * (ip - ipmin))/ (ipmax - ipmin));
}
```

**Comment by haps-basset** [comment]
Many thanks for your support.
I think the example code should be changed as follows:
mapValue(value, current,  (float)0.0, (float)100.0, (float)0.0, (float)2.0);
current = mapValue(value, (float)0.0, (float)100.0, (float)0.0, (float)2.0);
Overall, very nice example.
Once again many thanks

**Comment by Bodmer** [comment]
Ah, yes!

**Comment by Bodmer** [comment]
Fixed, raised version to 0.0.4 and created a release.
